src/test/java/com/gregtechceu/gtceu/api/recipe/OverclockLogicTest.java

```diff
@@ -114,6 +134,247 @@ private static BusHolder getBussesAndForm(GameTestHelper helper) {
         return new BusHolder(inputBus1, inputBus2, outputBus1, outputHatch1, controller);
     }
 
+    private static final Codec<HolderSet<Fluid>> FLUID_SET_CODEC = net.minecraft.core.RegistryCodecs
```

Comment:
All tests in this file should be moved, maybe to a seperate codec/serializer test file. The OverclockLogicTest should not end up edited. Sorry, I just threw them somewhere when debugging x3